src/LoginFunctionalityTest.java

```diff
@@ -0,0 +1,9 @@
+public class LoginFunctionalityTest {
+    public static void main(String[] args) {
+        System.out.println("navigate to log in page");
+        System.out.println("Enter username");
+        System.out.println("enter password");
+        System.out.println("verify login page");
```

Comment:
please try to add forgotten password tests

Reply:
added

Reply:
it should be reviewed and resolved by raised person  :) 
Look good